app/app.go

```diff
@@ -611,6 +628,19 @@ func (app *BinanceChain) initSlashing() {
 			DowntimeSlashFee:         10e8,
 		})
 	})
+	upgrade.Mgr.RegisterBeginBlocker(sdk.BEP159, func(ctx sdk.Context) {
+		// write slash params to beacon chain when upgrade to BEP159
+		app.slashKeeper.SetParams(ctx, slashing.Params{
```

Comment:
Do we need to add slash params, since we do not enable any slash mechenism in BEP159.

Reply:
Yes, we need this. Since `TooLowDelUnbondDuration` is in slash parameters, and it may be triggered by undelegate of BC validator.